src/routes/blog/posts/now-on-scoop/+page.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Introducing Scoop installation for Files
+description: Files is now available on Scoop! We are thrilled to announce that Files can now be easily installed via Scoop.
+thumbnail: /blog-resources/now-on-scoop/Hero.png
+date: 11/10/2024
+author: Lamparter
+---
+
+<script>
+  import { InfoBar } from "fluent-svelte";
+</script>
+
+Hello Files Community!
+
+I am thrilled to share some exciting news with you. As part of the ongoing efforts to make Files more accessible, the team has introduced Scoop as an additional method to install Files on your device.
+This means you can now install Files using Scoop, a command-line installer for Windows, known well for its simplicity and efficiency. 🚀
```

Comment:
I feel like the last sentence is a bit redundant. While I understand the intention to make the blog post a bit longer, I personally think it would be best to remove that part.

Reply:
Done in [`35d5c34`](https://github.com/Lamparter/FilesSite/commit/35d5c34e9314e445af00905818e0838799401887)